tasks/lib/jscs.js

```diff
@@ -138,7 +151,7 @@ exports.init = function( grunt ) {
      * Get inline options
      * @return {Object}
      */
-    JSCS.prototype.getOptions = function() {
+    JSCS.prototype.getInlineOptions = function() {
```

Comment:
Do you know of someone who uses grunt-jscs as a lib for this to be a breaking change?